vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -316,76 +330,48 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
-    //  2. Your AddressComplete account key must be defined as "window.addressCompleteKey".
+    // If you want to use this component with the Canada Post AddressComplete service:
+    // 1. The AddressComplete JavaScript script (and stylesheet) must be loaded.
+    // 2. Your AddressComplete account key must be defined.
     const pca = window['pca']
     const key = window['addressCompleteKey']
     if (!pca || !key) {
+      // eslint-disable-next-line no-console
+      console.log('AddressComplete not initialized due to missing script and/or key')
       return
     }
 
-    // Sets the id for the two form elements that are used by the AddressComplete code. If necessary this removes the
-    // id from previous elements.
-    this.moveElementId('street-address')
-    this.moveElementId('address-country-pca')
-
     // Destroy the old object if it exists, and create a new one.
     if (window['currentAddressComplete']) {
       window['currentAddressComplete'].destroy()
     }
     window['currentAddressComplete'] = this.createAddressComplete(pca, key)
   }
 
-  /**
-   * Sets the id attribute of the _named_ element to its name. If there was a pre-existing element with the id already
-   * set, it will be unset.
-   *
-   * @param name the name of the element for which to set the id.
-   */
-  private moveElementId (name: string): void {
-    const oldElement = document.getElementById(name)
-    const thisElement = this.$el.querySelector('[name="' + name + '"]')
-
-    // If it's already set, don't do it again.
-    if (oldElement !== thisElement) {
-      if (oldElement) {
-        oldElement.id = ''
-      }
-
-      thisElement.id = name
-    }
-  }
-
   /**
    * Creates the AddressComplete object for this instance of the component.
-   *
-   * @param pca the Postal Code Anywhere object provided by AddressComplete.
-   * @param key the key for the Canada Post account that is to be charged for lookups.
-   *
-   * @return an object that is a pca.Address instance.
+   * @param pca the Postal Code Anywhere object provided by AddressComplete
+   * @param key the key for the Canada Post account that is to be charged for lookups
+   * @returns an object that is a pca.Address instance
    */
   private createAddressComplete (pca, key: string): object {
     // Set up the two fields that AddressComplete will use for input.
     // Ref: https://www.canadapost.ca/pca/support/guides/advanced
     // Note: Use special field for country, which user can't click, and which AC will overwrite
     //       but that we don't care about.
-    //       (Option `populate: false` doesn't seem to work.)
     const fields = [
-      { element: 'street-address', mode: pca.fieldMode.SEARCH },
-      { element: 'address-country-pca', mode: pca.fieldMode.COUNTRY }
+      { element: this.streetAddressId, field: 'Line1', mode: pca.fieldMode.SEARCH },
+      { element: this.addressCountryId, field: 'CountryName', mode: pca.fieldMode.COUNTRY }
```

Comment:
I added "field" to work with latest AddressComplete library.

Also this is where we tell AddressComplete about the elements to use (using unique ids!).